sigs.yaml

```diff
@@ -2144,3 +2144,30 @@ workinggroups:
       tz: "PT (Pacific Time)"
       frequency: weekly
       url: https://docs.google.com/document/d/1RbC4SBZBlKth7IjYv_NaEpnmLGwMJ0ElpUOmsG-bdRA/edit
+  - name: K8s Infra
+    dir: wg-k8s-infra
+    mission_statement: >
+      A Working Group dedicated to migrating Kubernetes project infrastructure
+      over to the CNCF, and the creation of teams and processes for ongoing
+      maintenance. Involves collaboration with multiple SIGs such as
+      Architecture, Contributor Experience, Release, and Testing.
+    charter_link: charter.md
+    leadership:
+      chairs:
+      - name: Davanum Srinivas
+        github: dims
+        company: Huawei
+      - name: Aaron Crickenberger
+        github: spiffxp
+        company: Google
+    meetings:
+    - description: Regular WG Meeting
+      day: Friday
+      time: "9:00"
```

Comment:
Is this set?  @dims was mentioning trying to find another time.

Reply:
I'm happy to change this post-Doodle poll, either in this PR or a followup depending on when we resolve